addons/config/expected/csi.tanzu.vmware.com/v1alpha1/azurediskcsiconfig/default.yaml

```diff
@@ -7,5 +7,5 @@ metadata:
     tkg.tanzu.vmware.com/template-config: "true"
 spec:
   azureDiskCSIDriver:
-    namespace: tkg-system
-    deploymentReplicas: 1
+    namespace: kube-system
+    deploymentReplicas: 3
```

Comment:
This will no longer work with a dev plan will it? Don't we need to make this change depending on the number of control plane replicas?

Reply:
@miclettej , what options are we giving in the UI?

@timmycarr how much do we care about the dev plans?

Reply:
@randomvariable Yes, You are right I'm looking for a way to differentiate the two of them